civicrm.install

```diff
@@ -22,13 +24,25 @@ function civicrm_install() {
 
   $installed = $setup->checkInstalled();
   if ($installed->isSettingInstalled() || $installed->isDatabaseInstalled()) {
-    drupal_set_message(t("CiviCRM appears to have already been installed. Skipping full installation."));
+    \Drupal::messenger()
+      ->addMessage(t("CiviCRM appears to have already been installed. Skipping full installation."));
   }
 
   $setup->installFiles();
   $setup->installDatabase();
 }
 
+/**
+ * Implements hook_uninstall().
+ */
+function civicrm_uninstall() {
+  /** @var \Civi\Setup $setup */
+  $setup = _civicrm_setup();
+
+  $setup->uninstallFiles();
```

Comment:
Does this still work when civicrm.settings.php isn't writable? I can test it out later if you haven't already.

Reply:
Haven't tested it when not writable.